quantize.py

```diff
@@ -12,7 +12,8 @@
 from sentencepiece import SentencePieceProcessor
 
 try:
-    from GPTQ import GenericGPTQRunner, InputRecorder, lm_eval
+    from GPTQ import GenericGPTQRunner, InputRecorder
+    from eval import get_task_dict, evaluate
 except:
     pass
```

Comment:
can we remove this try/except as well? it will have easier to debug in the future

Reply:
it errors without that, there's a circular import that was added when it was moved to gpt-fast that's been there since the start, this more or less fixes it

Reply:
I see, is it possible to remove the circular import?

Reply:
probably i'll add it to the BE doc for gpt-fast, but its unrelated to the lm_eval stuff